src/Command/Traits/Json/NxXxSubcommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Predis\Command\Traits\Json;
+
+use Predis\Command\Command;
+use UnexpectedValueException;
+
+/**
+ * @mixin Command
+ */
+trait NxXxSubcommand
```

Comment:
NX/XX can be called option or argument or, maybe, parameter; but not sub-command.

Reply:
According to Redis specification for JSON.SET:

_Optional subcommands NX and XX modify this behavior for both new RedisJSON data type keys as well as the JSON Object keys in them._

Reply:
The documentation must be wrong. I have opened https://github.com/RedisJSON/RedisJSON/pull/900. Please at least wait for that PR to be resolved.

@chayim